src/test/java/io/confluent/connect/elasticsearch/helper/ElasticsearchHelperClient.java

```diff
@@ -49,17 +54,30 @@ public ElasticsearchHelperClient(String url, ElasticsearchSinkConnectorConfig co
     );
   }
 
-  public void deleteIndex(String index) throws IOException {
+  public void deleteIndex(String index, boolean isDataStream) throws IOException {
```

Comment:
maybe just have a `deleteDataStream()` method instead?

Reply:
In that case, should I add a public method called `deleteIndexOrDatastream()`? Otherwise, I would have to add an if/else block wherever `deleteIndex()` is currently called, which is a bit repetitive.